src/libpgmoneta/configuration.c

```diff
@@ -2032,6 +2038,1038 @@ pgmoneta_reload_configuration(bool* restart)
+      }
+      else if (!strcmp(key, "backup_max_rate"))
+      {
+         if (strlen(section) > 0)
+         {
+            if (as_int(config_value, &config->servers[server_index].backup_max_rate))
+            {
+               unknown = true;
+            }
+            pgmoneta_json_put(server_j, key, (uintptr_t)config->servers[server_index].backup_max_rate, ValueInt32);
+            pgmoneta_json_put(response, config->servers[server_index].name, (uintptr_t)server_j, ValueJSON);
+         }
+         else
+         {
+            if (as_int(config_value, &config->backup_max_rate))
+            {
+               unknown = true;
+            }
+            pgmoneta_json_put(response, key, (uintptr_t)config->backup_max_rate, ValueInt32);
+         }
+      }
+      else if (!strcmp(key, "network_max_rate"))
+      {
+         if (strlen(section) > 0)
+         {
+            if (as_int(config_value, &config->servers[server_index].network_max_rate))
+            {
+               unknown = true;
+            }
+            pgmoneta_json_put(server_j, key, (uintptr_t)config->servers[server_index].network_max_rate, ValueInt32);
+            pgmoneta_json_put(response, config->servers[server_index].name, (uintptr_t)server_j, ValueJSON);
+         }
+         else
+         {
+            if (as_int(config_value, &config->network_max_rate))
+            {
+               unknown = true;
+            }
+            pgmoneta_json_put(response, key, (uintptr_t)config->network_max_rate, ValueInt32);
+         }
+      }
+      else if (!strcmp(key, "manifest"))
+      {
+         if (strlen(section) > 0)
+         {
+            config->servers[server_index].manifest = pgmoneta_get_hash_algorithm(config_value);
+            pgmoneta_json_put(server_j, key, (uintptr_t)config->servers[server_index].manifest, ValueInt32);
+            pgmoneta_json_put(response, config->servers[server_index].name, (uintptr_t)server_j, ValueJSON);
+         }
+         else
+         {
+            config->manifest = pgmoneta_get_hash_algorithm(config_value);
+            pgmoneta_json_put(response, key, (uintptr_t)config->manifest, ValueInt32);
+         }
+      }
+      else
+      {
+         unknown = true;
+      }
+
+      if (unknown)
+      {
+         goto error;
+      }
+   }
+
+   end_time = time(NULL);
+
+   if (pgmoneta_management_response_ok(NULL, client_fd, start_time, end_time, compression, encryption, payload))
+   {
+      pgmoneta_management_response_error(NULL, client_fd, NULL, MANAGEMENT_ERROR_STATUS_NETWORK, compression, encryption, payload);
+      pgmoneta_log_error("Conf Set: Error sending response");
+
+      goto error;
+   }
+
+   elapsed = pgmoneta_get_timestamp_string(start_time, end_time, &total_seconds);
+
+   pgmoneta_log_info("Conf Set (Elapsed: %s)", elapsed);
+
+   pgmoneta_json_destroy(payload);
+
+   pgmoneta_disconnect(client_fd);
+
+   pgmoneta_stop_logging();
+
+   exit(0);
+error:
+   pgmoneta_log_info("Inside main error");
```

Comment:
This need to be an `pgmoneta_log_error` and need to have context of why it failed